ci/travis/install-dependencies.sh

```diff
@@ -216,7 +216,16 @@ install_dependencies() {
       # These packages aren't Windows-compatible
       pip_packages+=(blist)  # https://github.com/DanielStutzbach/blist/issues/81#issue-391460716
     fi
-    CC=gcc pip install "${pip_packages[@]}"
+
+    # Try n times; we often encounter OpenSSL.SSL.WantReadError (or others)
+    # that break the entire CI job: Simply retry installation in this case
+    # after n seconds.
+    while [ 1 ]
+    do
+      CC=gcc pip install "${pip_packages[@]}" && break;
```

Comment:
I think you want to have `n` somewhere here, or this will keep on going any number of iterations :)